### ISSUE TITLE
Multi-line editor indents wrapped lines when prepending spaces to text

### ISSUE THREAD

**Issue by belzecue** [godotengine/godot#56197]
### Godot version

3.4.1 stable

### System information

Linux, Ubuntu 20.04, Intel graphics

### Issue description

The MULTILINE hint for multi-line text entry is handled incorrectly in the editor.  When a line wraps due to insufficient space, if you go to the start of the line and add spaces, the entire block is indented instead of just the first line of the multi-line wrapped text.  You will see the entire text block get indented instead of the spaces just pushing out the first line.

If you open up the full text editor popup window, same thing happens once the text block starts wrapping.

Expected behavior when prepending space chars to this text is like any other text editor: the spaces get added and only the first line of the wrapped text block gets indented.

The text string itself remains correct, with no actual whitespace getting added to subsequent indented lines.  Seems this is only an issue with displaying the text in the control.

### Steps to reproduce

To repro, add a script to a node:

```
extends Node

export(String, MULTILINE) var text_stream: String

```

Go to the text editor for this property, enter a long line of text that flows/wraps across multiple lines, then add some spaces to the start of the text. 

https://user-images.githubusercontent.com/1931303/147263354-8030931b-25e7-4466-9a0e-9654577c4e22.mp4

### Minimal reproduction project

As described above.

**Comment by belzecue** [comment]
It strikes me that this might be intended behavior somehow, and that the expectation is, if you are entering spaces as the start of your text string then you want it to be treated differently than text starting with a non-whitespace char.  My use case is, I want to have two spaces at the start of my text string.  This formatting issue does not prevent me from doing that -- I was just surprised to see it indenting the entire block of wrapped text.

**Comment by belzecue** [comment]
3.0 stable to 3.0.6 has a cruder version of multi-line editing, where including the MULTILINE hint for EXPORT would add a UI button to pop open the full text editor.  This early version of the pop-up editor did not perform text wrap.  Long lines simply scrolled to the right.

3.1 stable introduced the multi-line wrapping we are accustomed to, and exhibits the same space-indent behaviour.  So this behavior has been there since the multi-line wrapping was added to the inspector property editor.

**Comment by kitbdev** [comment]
Fixed in 4.0 when wrapped line behavior changed.

- Related, indenting wrapped lines was recently properly re-introduced as optional in https://github.com/godotengine/godot/pull/88546
